sos/report/plugins/stunnel.py

```diff
@@ -0,0 +1,37 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.report.plugins import Plugin, IndependentPlugin
+
+
+class Stunnel(Plugin, IndependentPlugin):
+
+    short_desc = 'Universal SSL Tunnel'
+    plugin_name = "stunnel"
+    profiles = ('network', 'security')
+
+    packages = ('stunnel',)
+
+    def setup(self):
+
+        self.add_cmd_output([
+            "systemctl status stunnel*",
+            "journalctl --no-pager --unit stunnel*"
+        ])
```

Comment:
We actually have 2 wrappers for these. `Plugin.add_service_status()` and `Plugin.add_journal()`. The benefits of using these over `add_cmd_output()` are that we have standardized the collection patterns; for example with `add_journal()` you get more control over how the journal is collected, and it also uses a different size limit than typical collections.

With `add_service_status()` it (can) account for different init systems.